capturable/src/test/java/dev/shreyaspatil/capturable/controller/CaptureControllerTest.kt

```diff
@@ -55,6 +58,25 @@ class CaptureControllerTest {
         assertEquals(captureRequest.imageBitmapDeferred.isActive, true)
     }
 
+    @Test
+    fun captureRequests_collectedMultipleTimes() = runTest {
+        // Eagerly collect flow and cancel it to make sure we're subscribed to at least once.
+        asyncOnUnconfinedDispatcher { getRecentCaptureRequest() }.cancelAndJoin()
+
+        // When: Flow is collected again
+        asyncOnUnconfinedDispatcher {
+            try {
+                getRecentCaptureRequest()
+            } catch (e: Throwable) {
+                if (e !is CancellationException) {
+                    // Then: Make sure there was no crash
+                    fail(e.message)
+                }
+            }
+            Unit
```

Comment:
Can we just have `getRecentCaptureRequest()` again? because if crash occurs, anyway this will fail

Reply:
Sure. Do you want to add it to avoid using `Unit` or is there another reason? Asking for my own understanding.

Reply:
Because this just looked redundant to have try-catch in test cases

Reply:
Ahhh, sorry. You meant instead of the whole `try/catch` block. I did because I find it better to be explicit about things like this in test but if you find it redundant I'll change it.

Reply:
Changed in: [d7bc4fa](https://github.com/PatilShreyas/Capturable/pull/232/commits/d7bc4fa9e4c723f38f0c308641f4b486a375a5af)